docs/appendices/release-notes/5.6.3.rst

```diff
@@ -47,5 +47,21 @@ See the :ref:`version_5.6.0` release notes for a full list of changes in the
 Fixes
 =====
 
+- Fixed an issue with PostgreSQL style :ref:`INTERVALs <type-interval>`, which
+  caused days to be ignored if weeks are also present in the string, e.g.::
```

Comment:
```suggestion
  caused weeks to be ignored if weeks are also present in the string, e.g.::
```

See:
```
cr> SELECT '1 year 2 weeks 3 days'::INTERVAL;
+------------------------+
| 'P1Y3D'::interval      |
+------------------------+
| 1 year 3 days 00:00:00 |
+------------------------+
```

Reply:
yes, thx so much for double checking, I mixed up the desrcription.